sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/Transforms.java

```diff
@@ -186,23 +188,23 @@ private Transforms() {
      *
      * @param readResultItem The per page text extraction item result returned by the service.
      * @param pageNumber The page number.
-     *
      * @return A list of {@code FormSelectionMark}.
      */
     static List<FormSelectionMark> getReadResultFormSelectionMarks(ReadResult readResultItem, int pageNumber) {
         return readResultItem.getSelectionMarks().stream()
             .map(selectionMark -> {
                 final FormSelectionMark formSelectionMark = new FormSelectionMark(
                     null, toBoundingBox(selectionMark.getBoundingBox()), pageNumber);
-                final SelectionMarkState selectionMarkStateImpl = selectionMark.getState();
+                final com.azure.ai.formrecognizer.implementation.models.SelectionMarkState selectionMarkStateImpl
+                    = selectionMark.getState();
                 com.azure.ai.formrecognizer.models.SelectionMarkState selectionMarkState;
-                if (SelectionMarkState.SELECTED.equals(selectionMarkStateImpl)) {
+                if (SelectionMarkState.SELECTED.toString().equals(selectionMarkStateImpl.toString())) {
                     selectionMarkState = com.azure.ai.formrecognizer.models.SelectionMarkState.SELECTED;
-                } else if (SelectionMarkState.UNSELECTED.equals(selectionMarkStateImpl)) {
+                } else if (SelectionMarkState.UNSELECTED.toString().equals(selectionMarkStateImpl.toString())) {
                     selectionMarkState = com.azure.ai.formrecognizer.models.SelectionMarkState.UNSELECTED;
                 } else {
                     throw LOGGER.logThrowableAsError(new RuntimeException(
-                            String.format("%s, unsupported selection mark state.", selectionMarkStateImpl)));
+                        String.format("%s, unsupported selection mark state.", selectionMarkStateImpl)));
```

Comment:
nit: Since this is a really simple `String.format` we could drop that and just use string concatenation and save on the performance hit of `String.format`